data-generation/src/main/scala/thesis/LandyConsumer.scala

```diff
@@ -0,0 +1,72 @@
+package thesis
+
+import org.apache.spark.SparkContext
+import org.apache.spark.graphx.{Edge, Graph}
+import org.apache.spark.rdd.RDD
+import thesis.Action.{CREATE, UPDATE}
+import utils.{LogUtils, UtilsUtils}
+
+import java.time.Instant
+import scala.collection.mutable
+
+object LandyConsumer {
```

Comment:
Tror også at dette burde vært en companion object til Landy. Men er ikke så viktig. Da kunne du hatt en `apply`-funksjon som gjør at man kan skrive `Landy(logs)` istedet for `Landy.consume(logs)` siden det er jo den eneste måten å lage et Landy objekt.

Reply:
Aha, typ
```scala
class Landy(....)
object Landy(...) {
    def apply(logs): Landy = {
        ...
    }
}
```
?

Reply:
✅